src/test/java/api/loans/RenewalAPITests.java

```diff
@@ -152,7 +152,7 @@ public void canRenewRollingLoanFromCurrentDueDate()
 
     assertThat("due date should be 2 months after initial due date date",
       renewedLoan.getString("dueDate"),
-        isEquivalentTo(new DateTime(2018, 7, 12, 11, 21, 43)));
+      isEquivalentTo(new DateTime(2018, 7, 12, 11, 21, 43)));
```

Comment:
```suggestion
      isEquivalentTo(new DateTime(2018, 7, 12, 11, 21, 43, DateTimeZone.UTC)));
```

Reply:
Suggesting to add _DateTimeZone.UTC_ for all "new DateTime(int...)"

Reply:
Added DateTimeZone.UTC to all DateTime instantiations in this class